src/common.h

```diff
@@ -105,10 +105,10 @@ void ly_log_location(const struct lysc_node *scnode, const struct lyd_node *dnod
 /**
  * @brief Revert the specific logger's location data by number of changes made by ::ly_log_location().
  *
- * @param[in] scnode_steps Number of items in ::log_location.scnodes to forget.
- * @param[in] dnode_steps Number of items in ::log_location.dnodes to forget.
- * @param[in] path_steps Number of path strings in ::log_location.paths to forget.
- * @param[in] in_steps Number of input handlers ::log_location.inputs to forget.
+ * @param[in] scnode_steps Number of items in log_location_s.scnodes to forget.
```

Comment:
Please keep `::`, it simply means you require a link to an existing symbol and if it is not found, generate a warning.